lib/Alchemy/Phrasea/Controller/Prod/EditController.php

```diff
@@ -375,4 +378,144 @@ public function applyAction(Request $request) {
 
         return $this->app->json(['success' => true]);
     }
+
+
+    /**
+     * route GET "../prod/records/edit/presets/{preset_id}"
+     *
+     * @param Request $request
+     * @param $preset_id
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    public function presetsLoadAction($preset_id)
+    {
+        $ret = [];
+
+        if ( ($preset = $this->getPresetRepository()->find($preset_id)) === null) {
```

Comment:
I prefer `if (null === $preset = $this->getPresetRepository()->find($preset_id)) {}` which also fix PHPCS.